lms/templates/seq_module.html

```diff
@@ -32,6 +32,13 @@
     <button class="sequence-nav-button button-next">
         <span class="icon fa fa-chevron-next" aria-hidden="true"></span><span class="sr">${_('Next')}</span>
     </button>
+    % if masquerade_override:
+        <div class="wrapper-alert wrapper-alert-freestanding is-shown">
+            <div class="copy">
+                ${_("The content of this assignment is visible because you are masquerading; it is hidden to the student.")}
```

Comment:
@efischer19 Let's avoid the term "masquerading". 
How about: "This content is hidden from the learner." 
I don't feel that it's necessary to repeat the "You are now viewing the course as..." info because that is already showing in the bar at the top, do you agree?